migracja.rst

```diff
@@ -0,0 +1,235 @@
+**Laboratorium EE**
+
+*29 XII 2016*
+
+Model migracji z systemów EZD PUW i eDok
+========================================
+
+Wstęp
+<<<<<
```

Comment:
Sposób oznaczenia nagłówków wydaje się niespójny z innymi plikami w ramach ``ezd-analizy-it``.